sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -51,6 +52,9 @@
 
 class LineSource(FileBasedSource):
 
+  @unittest.skipIf(sys.version_info[0] == 3, 'This test still needs to be '
```

Comment:
Let's introduce an environment variable `RUN_SKIPPED_PY3_TESTS`, and if it is set to 1, let's not skip the tests. In a separate PR we should also add a possibility to unskip Python 3 tests we have already skipped. This will make it easier to see how many tests we should still fix, and which tests are already passing.